### PR TITLE
introducing goreleaser also in furyctl

### PR DESCRIPTION
**Pull request by lzecca78**
As furyagent, i've created the same logic also here.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -3,98 +3,81 @@ kind: pipeline
...
-      CGO_ENABLED: 0    
+      GITHUB_TOKEN:
+        from_secret: GITHUB_TOKEN
+    secrets:
+      - GITHUB_TOKEN
     commands:
-      - go mod vendor
-      - go build -a -ldflags '-extldflags "-static"' -o bin/releases/furyctl-darwin-amd64 .
+      - goreleaser check
+      - goreleaser --debug
```
Comment (lnovara) [inline .drone.yml]: Do we really need the `--debug`?

Comment (lzecca78) [reply .drone.yml]: not really. But can help in case something will go wrong for  finding root cause.

Comment (lnovara) [reply .drone.yml]: 👍

**Comment by ralgozino** [comment]
Please @lzecca78  add the installation instructions using `brew` to the README :) thanks!

**Comment by angelbarrera92** [comment]
I've added the brew instructions. Merging the change

**Pull request merged**